torch/onnx/symbolic_opset9.py

```diff
@@ -4891,22 +4872,22 @@ def bernoulli(g: jit_utils.GraphContext, input, generator=None, out=None):
             "Bernoulli", "generator is not supported for bernoulli", input
         )
 
-    dtype = symbolic_helper._try_get_scalar_type(input)
-    if dtype is None:
+    try:
+        dtype = _type_utils.JitScalarType.from_value(input)
+    except errors.OnnxExporterError:
         return symbolic_helper._unimplemented(
             "Bernoulli", "input dtype not accessible", input
         )
```

Comment:
Use the UNDEFINED check?

Reply:
`_try_get_scalar_type` passes `UNDEFINED` under the hood and `return None` when `UNDEFINED` is detected inside